app/models/event.rb

```diff
@@ -3,6 +3,12 @@ class Event < ApplicationRecord
   validates :friend_id, presence: true
   validates :event_type, presence: true
   validate :future_date
+  
+  TYPES = ['birthday', 'friendaversary', 'divorce party', 'graduation', 'baby shower', 'wedding']
+
+  
+  validates :event_type, inclusion: { in: TYPES,
+    messae: "%{value} is not a valid event type" }
```

Comment:
is there any way we can add an 'other'/'create event' category? may be something to add to the 'nice to have' list, but will be lots of events from different cultures that we haven't thought of